app/javascript/dashboard/routes/dashboard/search/components/SearchResultSection.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <section>
+    <div class="label-container">
+      <p>{{ title }}</p>
+    </div>
+    <slot />
+    <div v-if="empty" class="empty">
+      <fluent-icon icon="info" size="24px" class="icon" />
+      <p class="empty-state__text">
+        {{ $t('SEARCH.EMPTY_STATE', { item: titleCase, query }) }}
+      </p>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      default: '',
+    },
+    empty: {
+      type: Boolean,
+      default: false,
+    },
+    query: {
+      type: String,
+      default: '',
+    },
+  },
+  computed: {
+    titleCase() {
+      return this.title.toLowerCase();
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.search-list {
+  list-style: none;
+  margin: 0;
+  padding: var(--spacing-normal) 0;
+}
+
+.label-container {
+  position: sticky;
+  top: 0;
+  padding: var(--space-small);
+  background-color: var(--s-25);
+  z-index: 50;
```

Comment:
Please use variables here.
```suggestion
```

Reply:
The z-index is added to prevent this.

![Screenshot2022-12-19 at 13 53 58](https://user-images.githubusercontent.com/15716057/208380333-77cf14fa-15ee-4b6d-a217-fecdd5e8625c.png)